app.go

```diff
@@ -210,12 +210,12 @@ func (app *appImpl) exec() error {
 		for _, a := range actions {
 			a = app.actionMngr.Decorate(a)
 			if err := a.EnsureLoaded(); err != nil {
-				fmt.Fprintf(os.Stdout, "[WARNING] Action %q was skipped because it has an incorrect definition:\n%v\n", a.ID, err)
+				fmt.Fprintf(os.Stdout, "[WARNING] Action %q was skipped because it has an incorrect definition:\n%v\n", a.GetID(), err)
 				continue
 			}
 			cmd, err := action.CobraImpl(a, app.Streams())
 			if err != nil {
-				fmt.Fprintf(os.Stdout, "[WARNING] Action %q was skipped:\n%v\n", a.ID, err)
+				fmt.Fprintf(os.Stdout, "[WARNING] Action %q was skipped:\n%v\n", a.GetID(), err)
 				continue
 			}
 			cmd.GroupID = ActionsGroup.ID
```

Comment:
Added relevant feedback for this PM in https://github.com/launchrctl/web/pull/50/files#r1629088200